angular/projects/researchdatabox/form/src/app/component/dropdown-input.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, Input } from '@angular/core';
+import { FormFieldBaseComponent, FormFieldCompMapEntry, FormFieldModel } from "@researchdatabox/portal-ng-common";
+import { DropdownInputComponentConfig } from '@researchdatabox/sails-ng-common';
+import { get as _get, isEmpty as _isEmpty, isUndefined as _isUndefined } from 'lodash-es';
+
+export class DropdownInputModel extends FormFieldModel<string | number | null> {
```

Comment:
The generic type here should be `DropdownModelValueType`. Same for the other Model class generic types, they should be the matching `...ValueType` types.